Sources/tuist/tuist.docc/Tutorials/Tuist-Tutorials.tutorial

```diff
@@ -0,0 +1,24 @@
+@Tutorials(name: "Tuist Tutorials") {
```

Comment:
Few pieces of feedback here:

- I noticed some redundancy in the URL. What about turning this one into `tutorials/tuist`
<img width="374" alt="image" src="https://github.com/tuist/tuist/assets/663605/7379f5c3-3086-43da-9d53-4c4ddfdeaa07">

- If this page is an umbrella for all Tuist tutorials, the "Meet Tuist" intro is a bit misleading because there are some advanced sections that I don't expect to see in tutorials to get started, like `Tuist Cloud Enterprise`. I'd recommend to extract Tuist Cloud and Tuist Cloud Enterprise from here and have two groups, one for Tuist (from beginner to advance) and another one for Tuist Cloud. What do you think?

Reply:
I see, valid point. Let me check on this.